code/datums/status_effects/debuffs.dm

```diff
@@ -536,7 +567,12 @@ obj/effect/temp_visual/curse/Initialize()
 /datum/status_effect/debuff/feintcd
 	id = "feintcd"
 	alert_type = /atom/movable/screen/alert/status_effect/debuff/feintcd
-	duration = 30 SECONDS
+	duration = 15 SECONDS
```

Comment:
unsure if you intended to lower the cooldown on feinting from 30 -> 15 with this

Reply:
unintended